tests/test_lint.py

```diff
@@ -34,6 +34,10 @@ def test_condition_table_is_parameter_free():
 
     assert lint.condition_table_is_parameter_free(condition_df) is True
 
+    condition_df.fixedParameter1 =  condition_df.fixedParameter1.values.astype(int)
+
+    assert lint.condition_table_is_parameter_free(condition_df) is True
```

Comment:
just a warning: I think codecov or codacy complained about assert statements. Using the unittest.assertEquals() etc. was what we did for pypesto then.

Reply:
Yes, i remember. Only using it in unittests, not in production code.